modules/aws-outposts-environment-info-oc.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-environment-info-oc_{context}"]
+= Obtaining information from your {product-title} cluster
+
+You can use the {oc-first} to obtain information from your {product-title} cluster.
+
+[TIP]
+====
+You might find it convenient to store some or all of these values as environment variables by using the `export` command.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. List the infrastructure ID for the cluster by running the following command. Retain this value.
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+. Obtain the Amazon Machine Image (AMI) ID by running the following commands:
+
+.. List the compute machine sets on your cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
+[source,text]
+----
+NAME                           DESIRED   CURRENT   READY   AVAILABLE   AGE
+<compute_machine_set_name_1>   1         1         1       1           55m
+<compute_machine_set_name_2>   1         1         1       1           55m
+----
+
+.. Display the custom resource (CR) details for one of the listed compute machine sets:
++
+[source,terminal]
+----
+$ oc describe machineset <compute_machine_set_name_1> -n openshift-machine-api
+----
+
+.. In the output, the AMI ID is the value of the `spec.template.spec.providerSpec.value.ami` parameter. Find and retain this value.
+
+// This method works, might look at refining it if there is time to do so.
+
+// This is the generally suggested way, but it requires knowing the install dir:
+// $ grep ami \
+//   <installation_directory>/openshift/99_openshift-cluster-api_worker-machineset-0.yaml \
+//   | tail -n1 | awk '{print$2}'
+// ----
+// +
+// where `<installation_directory>` is the directory in which the installation program creates files.
+
+// Another possible method to just get the value wanted: https://kubernetes.io/docs/reference/kubectl/jsonpath/
```

Comment:
This may be more succinct if you can give a one liner that users can copy/paste

Reply:
Yeah - if I have time to puzzle this out (or one of y'all do) before GA I'll update with a single command :+1: For now, at least the AMI value is in the providerspec so there is not too much scrolling up to get it.

Reply:
I think what you want is `oc get -o jsonpath='.spec.template.spec.providerSpec.value.ami' -n openshift-machine-api <machine>`

Reply:
awesome thank you!